parserasgeo/features/cross_section.py

```diff
@@ -514,6 +520,76 @@ def cut_line_ratio(self):
             last_pt = pt
 
         return cl_length/length
+    
+    def define_channel_n(self):
+        """
+        Checks if the bank stations are represented in Manning's n (i.e. if Manning's n is defined at the statiosn that correspond with the bank stations)
```

Comment:
Please spell check and state how n values are selected if n values need to be created at the bank stations